src/main/cpp/Robot.cpp

```diff
@@ -36,11 +36,11 @@ void Robot::DisabledPeriodic() {}
  * RobotContainer} class.
  */
 void Robot::AutonomousInit() {
-  m_autonomousCommand = m_container.GetAutonomousCommand();
+  /*m_autonomousCommand = m_container.GetAutonomousCommand();
```

Comment:
Usually you shouldn't commit any comment blocks unless its documentation.

Since we are using version control, you can just delete these lines, and if you ever need to use them in the future, you can go through the git history to obtain the original code